### DISCUSSION TITLE
Is guest entries suited for my use case?

### DISCUSSION THREAD

**Discussion by juliencharette** [duncanmcclean/guest-entries#44] [Help] [answered]
Good afternoon,

I am looking to build a system where registered non-admin users can create, update and delete **their own** entries.

- Every submitted entries must be reviewed by an administrator/publisher before going live.
- Everytime a user updates, the update must be reviewed by an administrator/publisher the "updates" go live.
- Everytime a user deletes, the deletion of content must be approved by an administrator/publisher before it gets removed from the site.

Is this something I can achieve with Guest Entries plugin? Basically, the site is a "content repository" where any registered users can contribute content but I need a certain level of management/workflow before stuff goes live.

Thank you.

**Comment by duncanmcclean** [comment, marked as answer]
Hey!

I don't think it is, unfortunately. 

You could probably handle the creation piece with this addon and make it so created entries are unpublished by default. But then updating/deleting approval, probably not. 

You might be best building your own controllers to handle the workflow you want.
